src/containers/Profile/Profile.jsx

```diff
@@ -0,0 +1,209 @@
+      };
+    });
+  };
+
+  render() {
+    const { activePill } = this.state;
+    return (
+      <div className="profile-page">
+        <Header />
+
+        <div className="wrapper">
+          <div
+            className="header header-filter"
+            style={{
+              backgroundImage: `url(${bg})`,
+              backgroundSize: 'cover',
+              backgroundPosition: 'top center'
+            }}
+          />
+
+          <div className="main main-raised">
+            <div className="profile-content">
+              <Grid>
+                <Row>
+                  <div className="profile">
+                    <div className="avatar">
+                      <img
+                        src={christian}
+                        alt="Circle"
+                        className="img-circle img-responsive img-raised"
+                      />
+                    </div>
+                    <div className="name">
+                      <h3 className="title">Christian Louboutin</h3>
+                      <h6>Designer</h6>
+                    </div>
+                  </div>
+                </Row>
+                <div className="description text-center">
+                  <p>
+                    An artist of considerable range, Chet Faker — the name taken
+                    by Melbourne-raised, Brooklyn-based Nick Murphy — writes,
+                    performs and records all of his own music, giving it a warm,
+                    intimate feel with a solid groove structure.{' '}
+                  </p>
+                </div>
+
+                <Row>
+                  <Col md={6} mdOffset={3}>
+                    <Tab.Container id="tabs" defaultActiveKey={activePill}>
+                      <div className="profile-tabs">
+                        <div className="nav-align-center">
+                          <Nav
+                            bsStyle="pills"
+                            className="nav-pills-primary"
+                            onSelect={this.handlePill}
+                          >
+                            <NavItem eventKey="studio">
+                              <i className="material-icons">camera</i>
+                              Studio
+                            </NavItem>
+                            <NavItem eventKey="work" title="Item">
+                              <i className="material-icons">palette</i>
+                              Work
+                            </NavItem>
+                            <NavItem eventKey="shows">
+                              <i className="material-icons">favorite</i>
+                              Favorite
+                            </NavItem>
+                          </Nav>
+
+                          <Tab.Content animation={false} className="gallery">
+                            <Tab.Pane eventKey="studio">
+                              <Row>
+                                <Col md={6}>
+                                  <img
```

Comment:
for the image elements we can use the `<Image />` component from [react-bootstrap](https://react-bootstrap.github.io/components.html#images)

```
<Image 
  src={chris1} 
  rounded
  responsive
/>
```